core/src/OpenMPTarget/Kokkos_OpenMPTarget_Parallel.hpp

```diff
@@ -56,6 +56,377 @@
 namespace Kokkos {
 namespace Impl {
 
+template <class Reducer>
+struct OpenMPTargetReducerWrapper {
+  typedef typename Reducer::value_type value_type;
+
+  KOKKOS_INLINE_FUNCTION
+  static void join(value_type& dest, const value_type& src) {
```

Comment:
Don't you get warning from `dest` and `src` not being used?

Reply:
Yeah technically if you get a warning from this thing you got bigger issues (like that you use an unimplemented code path ;-) ). Not sure if we should spend too much time cleaning warnings now. In particular since OpenMPTarget stuff throws tons of warnings we can't avoid right now ...

Reply:
addressed